front/databaseinstance.form.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * ---------------------------------------------------------------------
+ * GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2015-2021 Teclib' and contributors.
+ *
+ * http://glpi-project.org
+ *
+ * based on GLPI - Gestionnaire Libre de Parc Informatique
+ * Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ *
+ * ---------------------------------------------------------------------
+ *
+ * LICENSE
+ *
+ * This file is part of GLPI.
+ *
+ * GLPI is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * GLPI is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ * ---------------------------------------------------------------------
+ */
+
+use Glpi\Event;
+
+include ('../inc/includes.php');
+
+Session::checkCentralAccess();
+
+if (!isset($_GET["id"])) {
+   $_GET["id"] = "";
+}
+
+if (!isset($_GET["databases_id"])) {
+   $_GET["databases_id"] = "";
+}
+
+$instance = new DatabaseInstance();
+if (isset($_POST["add"])) {
+   $instance->check(-1, CREATE, $_POST);
```

Comment:
I get a `You don't have permission to perform this action.` when trying to add a new DB instance.

Reply:
Strange... Maybe you/I forget to disconnect; I'll double check

Reply:
Indeed, it is OK.